assets/js/vue/FileManager.vue

```diff
@@ -61,24 +61,25 @@
             shading-color="rgba(0,0,0,0.4)"
         >
             <template>
-                <p>
-                    Uploading {{ humanSize(doneFileSize) }} of {{ humanSize(totalFileSize) }}
+                <p style="text-align: center">
+                    Uploaded {{ humanSize(doneFileSize) }} of {{ humanSize(totalFileSize) }}
                 </p>
-                <p>
-                    Uploading file {{ doneFiles }} of {{ totalFiles }}
+                <p style="text-align: center">
+                    Uploaded file {{ doneFiles }} of {{ totalFiles }}
                 </p>
                 <DxProgressBar
                   id="progress-bar-status"
                   :min="0"
                   :max="totalFileSize"
                   :value="doneFileSize"
-                  width="90%"
+                  width="100%"
                 />
                 <DxButton
-                    text="Cancel Upload!"
+                    text="Cancel Upload"
                     type="danger"
                     styling-mode="contained"
                     @click="stopProcess"
+                    :element-attr="cancelUploadBtn"
                  />
                  <p>
                     If the filesize is large? Click Cancel!
```

Comment:
Should we change this text?

Reply:
Just saw that comment from William, will do it